src/matrixrtc/MatrixRTCSessonManager.ts

```diff
@@ -97,6 +100,18 @@ export class MatrixRTCSessionManager extends TypedEventEmitter<MatrixRTCSessionM
         return this.roomSessions.get(room.roomId)!;
     }
 
+    private onTimeline = (event: MatrixEvent): void => {
+        if (event.getType() !== EventType.CallEncryptionPrefix) return;
+
+        const room = this.client.getRoom(event.getRoomId());
+        if (!room) {
+            logger.error(`Got room state event for unknown room ${event.getRoomId()}!`);
+            return;
+        }
+
+        this.getRoomSession(room).onCallEncryption(event);
```

Comment:
Do we want to create a new session for this event if one doesn't exist? If not, we probably want `getActiveRoomSession()`.

Reply:
:man_facepalming:

Reply:
Hmm, now that I am thinking about it, perhaps we want? In case the event arrives before state?

Reply:
Yeah, agreed - if we're not checking for existing events then we probably need to be capturing them whenever they arrive, even if there's no current call. I suppose it would only happen if someone sends a key before their call membership event, but that could easily happen. This does seem like the easier way of doing it.